server/src/main/java/org/opensearch/search/SearchService.java

```diff
@@ -298,17 +298,17 @@ public class SearchService extends AbstractLifecycleComponent implements IndexEv
     // settings to configure maximum slice created per search request using OS custom slice computation mechanism. Default lucene
     // mechanism will not be used if this setting is set with value > 0
     public static final String CONCURRENT_SEGMENT_SEARCH_TARGET_MAX_SLICE_COUNT_KEY = "search.concurrent.max_slice_count";
```

Comment:
same for this rename `CONCURRENT_SEGMENT_SEARCH_TARGET_MAX_SLICE_COUNT_KEY` to `CONCURRENT_SEGMENT_SEARCH_MAX_SLICE_COUNT_KEY`